test/Microsoft.Azure.Devices.Edge.Test/PriorityQueues.cs

```diff
@@ -54,7 +55,7 @@ public async Task PriorityQueueModuleToModuleMessages()
         }
 
         [Test]
-        [Category("Unstable")]
+        [Category("UnstableOnArm")]
```

Comment:
Why is it unstable on arm?

Reply:
All the trc tests are unstable on arm32. Gets some resource unavailable exception. Haven't dove deep on it